lib/relay.js

```diff
@@ -150,14 +150,23 @@ function requestHandler(filterRules) {
             .status(ioResponse.status)
             .set(ioResponse.headers);
 
-          // keep chunked http requests without content-length header
-          const isChunked =
-            undefsafe(ioResponse, 'headers.transfer-encoding') === 'chunked';
-          if (isChunked) {
-            httpResponse.write(ioResponse.body);
-            httpResponse.end();
-          } else {
-            httpResponse.send(ioResponse.body);
+          const encodingType = undefsafe(
+            ioResponse,
+            'headers.transfer-encoding',
+          );
+          try {
+            // keep chunked http requests without content-length header
+            if (encodingType === 'chunked') {
+              httpResponse.write(ioResponse.body);
+              httpResponse.end();
+            } else {
+              httpResponse.send(ioResponse.body);
+            }
+          } catch (err) {
+            logger.error(
+              { ...logContext, encodingType, err },
+              'Unsupported encoding type',
```

Comment:
We don't know what causes the error, this should just be something like  `'error forwarding response`